### PR TITLE
fix: grammar patch

### PR DESCRIPTION
**Pull request by esavier**

### CODE REVIEW

File: docs/rfc/0022_Materialization_Types_01.md [hunk 1]
```diff
@@ -420,17 +420,17 @@ If in the field value `null` is present, we have to assign most generic type in
 # Decision
 
 Document was changed to reflect decisions made during a meeting (23.08.2021) (mostly TLDR section).
-We are providing support for basic numeric types, strings and booleans. We support nesting these values in objects.
+We are providing support for basic numeric types, strings and boolean. We support nesting these values in objects.
```
Comment (luke-biel) [inline docs/rfc/0022_Materialization_Types_01.md:423]: https://en.wiktionary.org/wiki/boolean

File: docs/rfc/0022_Materialization_Types_01.md [hunk 2]
```diff
@@ -420,17 +420,17 @@ If in the field value `null` is present, we have to assign most generic type in
...
 
 We have moved arrays to later phase of development, their support requires additional RFC.
-Same goes for nullability, everything, at this moment is non-nullable, and changing this requires separate document.
-From discussion arisen type casting problem, when users would like to store numeric values as strings, for better precision handling.
-This also requires separate document and discussion.
+The same goes for nullability, everything, at this moment, is non-nullable, and changing this requires a separate document.
```
Comment (luke-biel) [inline docs/rfc/0022_Materialization_Types_01.md:426]:
```suggestion
The same goes for nullability, everything at this moment is non-nullable, and changing this requires a separate document.
```

File: docs/rfc/0022_Materialization_Types_01.md (hunk 1 above)
Comment (esavier) [reply docs/rfc/0022_Materialization_Types_01.md:423]: oki doki

File: docs/rfc/0022_Materialization_Types_01.md (hunk 2 above)
Comment (esavier) [reply docs/rfc/0022_Materialization_Types_01.md]: rewritten

**Comment by esavier** [comment]
okay, i messed up, closing

**Pull request closed without merging**